undefined_syms.ld

```diff
@@ -144,50 +113,50 @@ sMainMenuButton = 0x030055d8;
 gPauseMenu = 0x030055e0;
 
 /* Sound Library */
-D_030055f0 = 0x030055f0;
-D_030055f4 = 0x030055f4;
-D_03005600 = 0x03005600;
-D_03005610 = 0x03005610;
-D_03005620 = 0x03005620;
-D_0300562c = 0x0300562c;
-D_03005630 = 0x03005630;
-D_03005634 = 0x03005634;
-D_03005638 = 0x03005638;
-D_0300563c = 0x0300563c;
-D_03005640 = 0x03005640;
-D_03005644 = 0x03005644;
-D_03005648 = 0x03005648;
-D_03005650 = 0x03005650;
-D_030056a0 = 0x030056a0;
-D_03005720 = 0x03005720;
-D_03005b20 = 0x03005b20;
-D_03005b24 = 0x03005b24;
-D_03005b28 = 0x03005b28;
-D_03005b30 = 0x03005b30;
-D_03005b3c = 0x03005b3c;
+gMidiVCOUNTAtStart = 0x030055f0;
+gMidiSoundMode = 0x030055f4;
+gMidiRVB_Scratch = 0x03005600;
+gMidiSamplerCount = 0x03005610;
+gMidiEQ_Area = 0x03005620;
+gMidiPlayerNewDeltaTime = 0x0300562c;
+gMidiRVB_Control2 = 0x03005630;
+gMidiRVB_Control4 = 0x03005634;
+gMidiScratchSize = 0x03005638;
+gMidiPCMBufR = 0x0300563c;
+gMidiLFO_Depth = 0x03005640;
+gMidiLFO_Player = 0x03005644;
+gMidiCommVarCurrent = 0x03005648;
+gMidiNotePool = 0x03005650;
+gMidiPSGChannelPool = 0x030056a0;
+gMidiSampleTable = 0x03005720;
+gMidiCommVarTotal = 0x03005b20;
+gMidiPCMBufSize32 = 0x03005b24;
+gMidiEQ_HighGain = 0x03005b28;
+gMidiLFO = 0x03005b30;
+gMidiLFO_Mode = 0x03005b3c;
 D_03005b40 = 0x03005b40;
-D_03005b44 = 0x03005b44;
-D_03005b48 = 0x03005b48;
-D_03005b50 = 0x03005b50;
-D_03005b78 = 0x03005b78;
-D_03005b7c = 0x03005b7c;
-D_03005b80 = 0x03005b80;
-D_03005b84 = 0x03005b84;
-D_03005b88 = 0x03005b88;
-D_03005b8c = 0x03005b8c;
-D_03005b90 = 0x03005b90;
-D_03005b94 = 0x03005b94;
-D_03005ba0 = 0x03005ba0;
+gMidiEQ_IsGlobal = 0x03005b44;

```

Comment:
not a fan of random underscores in some of these variable names, considering other ones don't have them